pkg/coscheduling/core/core.go

```diff
@@ -191,16 +191,16 @@ func (pgMgr *PodGroupManager) PostBind(ctx context.Context, pod *corev1.Pod, nod
 	if pgCopy.Status.Phase != pg.Status.Phase {
 		pg, err := pgMgr.pgLister.PodGroups(pgCopy.Namespace).Get(pgCopy.Name)
 		if err != nil {
-			klog.Error(err)
+			klog.ErrorS(err, "pod group", klog.KObj(pgCopy))
 			return
 		}
 		patch, err := util.CreateMergePatch(pg, pgCopy)
 		if err != nil {
-			klog.Error(err)
+			klog.ErrorS(err, "pod group", klog.KObj(pg), "pod group", klog.KObj(pgCopy))
```

Comment:
- the ErrorS has a 2nd parameter to log the "msg"
- logging one pod group is good enough as `klog.KObj` just dumps its "namespace/name".

```suggestion
			klog.ErrorS(err, "Failed to create merge patch", "podGroup", klog.KObj(pg))
```